source/404.html.md.erb

```diff
@@ -0,0 +1,10 @@
+---
+title: "404"
```

Comment:
```suggestion
title: "Page not found – GOV.UK One Login Technical Documentation"
```
To match the [GOV.UK Design System](https://design-system.service.gov.uk/patterns/page-not-found-pages/) guidance :)

Reply:
@TobySaundersGDS This is the one I mentioned on Slack ^

Reply:
I must have reolved by mistake - I've added it now 👍🏻